bionic/flow.py

```diff
@@ -795,6 +795,69 @@ def then_set(self, name, value):
         return self
 
 
+class BlessedPath(object):
```

Comment:
Oh, BTW, I wonder if this class should live in some other module, like `util.py`?

Reply:
Yep, definitely should.  Moving!